python/cudf/cudf/tests/test_datetime.py

```diff
@@ -1297,3 +1297,27 @@ def test_is_leap_year():
     got2 = gIndex.is_leap_year
 
     assert_eq(expect2, got2)
+
+
+def test_is_month_start():
+    data = [
```

Comment:
generally prefer to `@pytest.parametrize` over input data even if there's only one test case, it makes it a little easier to add test cases later.

Reply:
ok, updated!